ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -21,6 +21,12 @@ No changes are required for existing deployments.
   - `node_type_keys.<key>.custom_templates_extra_vars` introduced in 3.7.0 has been removed. This is non-breaking since the templating environment
   used for custom templates now supports all host and group vars by default.
 
+#### Changes to eos_cli_config_gen role
+
+- **NTP authentication keys variables:**
+
+  - `authentication_key.key_type` introduced in 3.8.0 allow you to define the authentication key type.  If the key type is not defined, the previous behavior will be preserved.  The key type will be set to 7 as default starting in 4.0.0.
```

Comment:
Probably worth stating what the previous behavior is in this otherwise as a common user I don't know where to look for it.

Reply:
I think it's enough for this context - we will capture more info on this in 4.0 when behavior changes.